run.py

```diff
@@ -69,8 +69,14 @@ def run_one_step(func, nwarmup=WARMUP_ROUNDS, model_flops=None, num_iter=10, mod
             model_analyzer.set_export_csv_name(export_dcgm_metrics_file)
             model_analyzer.add_mem_throughput_metrics()
         model_analyzer.start_monitor()
-
-    for _i in range(num_iter):
+    if stress != 0:
+        cur_time = time.time_ns()
+        start_time = cur_time
+        target_time = stress * 60 * 1e9 + start_time
+        num_iter = -1
+    else:
+        _i = 0
+    while ( num_iter != -1 and _i < num_iter ) or ( stress != 0 and cur_time < target_time ) :
```

Comment:
If `stress` is not `0`, what will be the `_i` value here?

Reply:
If the `stress` is not 0, the `_i` will never be touched because `_i` follows `num_iter` and `num_iter == -1`. So we don't need to give it a value.

Reply:
fixed by https://github.com/pytorch/benchmark/pull/1002/commits/40efc6d4630c3098648f122e8abf03fdac60e111

Reply:
Let's make this predicate clearer:
```suggestion
    while (not stress and _i < num_iter ) or (stress and cur_time < target_time ) :
```